requirements.txt

```diff
@@ -1,14 +1,14 @@
 Cython==0.29.10
 keras==2.1.6
-matplotlib==3.1.1
+matplotlib==3.5.1
 natsort==6.0.0
 nose==1.3.7
 numpy==1.21.0
 opencv-python==4.2.0.32
-pandas==0.24.2
+pandas==1.1.3
```

Comment:
We had some issues with pandas 1.1.3. It has a bug that caused issues with reading the metadata CSV file:
https://github.com/pandas-dev/pandas/issues/37094

pandas 1.1.5 seems to work fine